modules/rhdh-plugins-reference/proc-setting-up-openshift-pipelines-to-download-or-view-sboms.adoc

```diff
@@ -0,0 +1,156 @@
+:_mod-docs-content-type: PROCEDURE
+[id="setting-up-openshift-pipelines-to-download-or-view-sboms_{context}"]
+
+= Setting up OpenShift Pipelines to view SBOM
+
+In Red Hat Developer Hub, under the CI tab, the Pipeline Runs section offers a feature to view Software Bill of Material (SBOM). This enhances transparency and control over your supply chain by listing all the software libraries a component utilizes. These libraries may provide specific functionality or facilitate development.
+
+You can use an SBOM to better understand the composition of your software, identify vulnerabilities, and assess the potential impact of any security issues that might arise.
+
+.Options to download or view SBOM
+image::rhdh-plugins-reference/sbom.png[]
+
+[NOTE]
+====
+The SBOM task discussed in this chapter uses `syft` for example purposes only. `syft` is a community project and is not supported by Red Hat Developer Hub by default. For guidance on implementing these or any other community-supported tasks consult your support agreements or contact support services.
+====
+
+.Prerequisites
+
+* You have link:https://docs.openshift.com/container-platform/4.14/web_console/web-console.html#web-console[logged in to the web console].
```

Comment:
Consider using the attribute:

```suggestion
* You have link:https://docs.openshift.com/container-platform/4.14/web_console/web-console.html#web-console[logged in to the {ocp-short} web console].
```

See the attribute list: https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/blob/main/artifacts/attributes.adoc